clio-server/config/docker-compose.yml.ctmpl

```diff
@@ -13,6 +17,22 @@ app:
     - /etc/localtime:/etc/localtime:ro
     - {{$app_dir}}/{{$app_conf}}:{{$conf_dir}}/{{$app_conf}}:ro
     - {{$app_dir}}/{{$logback_conf}}:{{$conf_dir}}/{{$logback_conf}}:ro
-    - /local/clio_logs:{{env "CLIO_LOG_DIR"}}
   restart: always
+  links:
+    - logger
+  depends_on:
+    - logger
+  volumes_from:
+    - logger
+
+logger:
+  image: broadinstitute/fluentd-gcp:latest
+  expose:
+    - 5140/tcp
+    - 5140/udp
```

Comment:
Do we need to expose these? They aren't listed in the [cromwell docker-compose config](https://github.com/broadinstitute/gotc/blob/develop/config/cromwell/docker-compose.yml.ctmpl#L56).